### ISSUE TITLE
conditional activity review

### ISSUE THREAD

**Issue by Iankimble** [makaiW/ap.python#4]
1. answer- correct
2. answer- partially correct
a conditional does not exclusively need to be true or false, we can evaluate other data types and return different scenarios. 
3. answer- partially correct
This is not the syntax we use in python for writing conditional statements
4. answer- incorrect
You didn't answer the question
5. answer- incorrect
You didn't answer the question

submission boost +1
partial credit boost
final score: 3/5

**Comment by makaiW** [comment]
yeah sorry my power went so my computer wasn't charging and it died before
I could finished

**Comment by makaiW** [comment]
and there is no 5
